app/src/main/java/com/devshish/internship/presentation/ui/home/HomeFragment.kt

```diff
@@ -46,6 +51,11 @@ class HomeFragment : Fragment(R.layout.fragment_home) {
             topTracks.observe(viewLifecycleOwner) { tracks ->
                 trackItemAdapter.submitList(tracks)
             }
+
+            isLayoutRefreshing.observe(viewLifecycleOwner) { isRefreshing ->
+                binding.layoutCharts.root.isVisible = !isRefreshing
```

Comment:
I think it is better to not hide layoutCharts , so remove this line, and then we can merge